receiver/googlecloudspannerreceiver/config.go

```diff
@@ -55,6 +56,11 @@ func (config *Config) Validate() error {
 			config.ID(), config.TopMetricsQueryMaxRows)
 	}
 
+	if config.CardinalityTotalLimit < 0 {
+		return fmt.Errorf("%v `cardinality_total_limit` must be not negative, current value is %vs",
```

Comment:
What's this `s` at the end of the format? Also, try to use double quotes instead of backticks here.

Reply:
Done. `s` - it was my mistake ((